pydicom/dataset.py

```diff
@@ -1415,7 +1415,7 @@ def __init__(self,
             True (default) if little-endian transfer syntax used; False if
             big-endian.
         """
-        Dataset.__init__(self, dataset)
+        super(FileDataset, self).__init__(dataset)
```

Comment:
While I also prefer that syntax, I don't see any difference in this case.
Also, I'm not sure what problem you are referring to:
```
Python 3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 18:11:49) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pydicom
>>> pydicom.Dataset.items
<function Dataset.items at 0x00000147CEF0D2F0>
>>> pydicom.FileDataset.items
<function Dataset.items at 0x00000147CEF0D2F0>
```

Reply:
Hm, are you sure you did the tests with the same version? 
In the last released version, there is indeed a bug that `Dataset` does not have `items` (l introduced this in the patch version...), but that is fixed in master, and there will be another patch release soon, I think. 
BTW, pydicom 1.2.0 does not have this problem, and the patch is only needed for Python 3.6.7 and 3.7.1. As you are using Python 3.6.6, you should be fine with that version.